spec/ruby/core/string/byterindex_spec.rb

```diff
@@ -0,0 +1,397 @@
+# -*- encoding: utf-8 -*-
+require_relative '../../spec_helper'
+require_relative 'fixtures/classes'
+require_relative 'fixtures/utf-8-encoding'
```

Comment:
minor: Looks like `fixtures/utf-8-encoding` is never used.

Reply:
Right, removed.